pkgs/top-level/all-packages.nix

```diff
@@ -31183,7 +31183,9 @@ in
 
   pgmanage = callPackage ../applications/misc/pgmanage { };
 
-  pgadmin = callPackage ../applications/misc/pgadmin {
+  pgadmin4 = callPackage ../tools/admin/pgadmin { };
+
+  pgadmin3 = callPackage ../applications/misc/pgadmin {
```

Comment:
Please move the files to one place and append version numbers to the directories.